build.gradle

```diff
@@ -49,6 +49,9 @@ ext {
 
     awsSdkVersion = '2.16.12'
     dependency = [
+        android: [
+            desugar_jdk_libs: 'com.android.tools:desugar_jdk_libs:1.0.5',
```

Comment:
For better or worse, I've avoided `_` in the other ones, so far.

How about just:
```suggestion
            desugartool: 'com.android.tools:desugar_jdk_libs:1.0.5',
```